lib/features/composer/presentation/composer_view.dart

```diff
@@ -141,42 +141,81 @@ class ComposerView extends GetWidget<ComposerController>
     return Container(
       padding: const EdgeInsets.symmetric(vertical: 16),
       color: Colors.white,
-      child: Row(mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            buildTextButton(
-                AppLocalizations.of(context).cancel,
-                textStyle: const TextStyle(fontWeight: FontWeight.w500, fontSize: 17, color: AppColor.lineItemListColor),
-                backgroundColor: AppColor.emailAddressChipColor,
-                width: 150,
-                height: 44,
-                radius: 10,
-                onTap: () => controller.closeComposer()),
-            const SizedBox(width: 12),
-            buildTextButton(
-                AppLocalizations.of(context).save_to_drafts,
-                textStyle: const TextStyle(fontWeight: FontWeight.w500, fontSize: 17, color: AppColor.colorTextButton),
-                backgroundColor: AppColor.emailAddressChipColor,
-                width: 150,
-                height: 44,
-                radius: 10,
-                onTap: () => controller.saveEmailAsDrafts(context)),
-            const SizedBox(width: 12),
-            buildTextButton(
-                AppLocalizations.of(context).send,
-                width: 150,
-                height: 44,
-                radius: 10,
-                onTap: () => controller.sendEmailAction(context)),
-          ]
+      child: Stack(
+        alignment: Alignment.centerRight,
+        children: [
+          Row(mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              buildTextButton(
+                  AppLocalizations.of(context).cancel,
+                  textStyle: const TextStyle(fontWeight: FontWeight.w500, fontSize: 17, color: AppColor.lineItemListColor),
+                  backgroundColor: AppColor.emailAddressChipColor,
+                  width: 150,
+                  height: 44,
+                  radius: 10,
+                  onTap: () => controller.closeComposer()),
+              const SizedBox(width: 12),
+              buildTextButton(
+                  AppLocalizations.of(context).save_to_drafts,
+                  textStyle: const TextStyle(fontWeight: FontWeight.w500, fontSize: 17, color: AppColor.colorTextButton),
+                  backgroundColor: AppColor.emailAddressChipColor,
+                  width: 150,
+                  height: 44,
+                  radius: 10,
+                  onTap: () => controller.saveEmailAsDrafts(context)),
+              const SizedBox(width: 12),
+              buildTextButton(
+                  AppLocalizations.of(context).send,
+                  width: 150,
+                  height: 44,
+                  radius: 10,
+                  onTap: () => controller.sendEmailAction(context)),
+            ]
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              buildIconWithUpperMenu(
+                const Icon(Icons.more_vert), 
```

Comment:
Should use icons according to the design. Please check if there is an icon in the `assets` folder. If not, `export` it from `figma`  https://www.figma.com/file/rRVxJpCu1G22D0jqyvK7I9/TeamMail?node-id=2%3A9